test/misc.jl

```diff
@@ -654,7 +654,8 @@ end
 
 include("testenv.jl")
 
-let flags = Cmd(filter(a->!occursin("depwarn", a), collect(test_exeflags)))
+
+let flags = Cmd(replace.(test_exeflags, r"^--depwarn=.*" => "--depwarn=yes"))
```

Comment:
This depends on `--depwarn` already occurring in test_exeflags; it seems more explicit to filter out --depwarn arguments and then add --depwarn=yes.

Reply:
bother, it does.
Too bad as I find this much easier to read.